src/components/StatusButton.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import {
+  GestureResponderEvent,
+  Pressable,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+import Svg, { Circle, Path } from 'react-native-svg';
+
+type Param = {
+  title: string;
+  status: number;
+  onPress: (event: GestureResponderEvent) => void;
+};
```

Comment:
ogólnie fajnie jakby button przyjmował wszystkie propsy jakie normalnie przyjmuje komponent `<Pressable />`

Zrób to w taki sposób:
`type Props = React.ComponentPropsWithRef<typeof Pressable> & { jakieś propsy, których pressable nie przyjmuje, a potrzebujemy je w komponencie }`